### PR TITLE
fix file system not refresh on exFAT

### LINKED ISSUE #28268: Godot can not import files on exFAT drive
**Godot version:**
3.1 stable (tested on mono & classic)

**OS/device including version:**
Windows 10

**Issue description:**
Godot engine is portable, which is really an advantage. I use a fast external SSD (Samsung SSD 860 EVO 2.5" (1 To)), exFat formatted. The folder with Godot projects is marked as indexed in Windows indexation settings.

Problem is Godot do not refresh it's own file indexing when I try to import new files (no matter which, images, dae models, ogg sounds, scripts and so on). It sees them only when I restart the editor. It also does not matter if I started it from the file link, project link, or from ```godot -e```.
When I update files (for example, save a new image to an already imported one), godot updates it immediately.

The problem is both on mono and classic versions. If I create a project on a main drive (which is an NTFS Samsung 850 EVO 250 gb), there are no problems, it imports everything immediately.
Building and exporting work without problems on all drives.

**Steps to reproduce:**
Create a blank project on external SSD and try to import files.

### PR DESCRIPTION
**Pull request by kpkhxlgy0**
fix file system not refresh on exFAT

*Bugsquad edit:* Fixes #28268.

### CODE REVIEW

File: editor/editor_file_system.cpp [hunk 1]
```diff
@@ -2142,6 +2142,8 @@ EditorFileSystem::EditorFileSystem() {
 	}
 	//this should probably also work on Unix and use the string it returns for FAT32
 	using_fat_32 = da->get_filesystem_type() == "FAT32";
+	//this should probably also work on Unix and use the string it returns for exFAT
+	using_ex_fat = da->get_filesystem_type() == "exFAT";
```
Comment (akien-mga) [inline editor/editor_file_system.cpp]: Changes are fine, but IMO it's not necessary to have two booleans enabling the same features. So I'd just do
```
using_fat_32 = (da->get_filesystem_type() == "FAT32" || da->get_filesystem_type() == "exFAT");
```

Comment (akien-mga) [reply editor/editor_file_system.cpp]: Could you amend the commit to do this change?

Comment (Calinou) [reply editor/editor_file_system.cpp]: @kpkhxlgy0 Bump :slightly_smiling_face:

Comment (akien-mga) [reply editor/editor_file_system.cpp]: I force-pushed the requested changes.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by kpkhxlgy0** [comment]
Sorry, I am a little busy recently, didn't log in to github. Thanks for merge my pr.

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**